src/ws/service/database_handlers.rs

```diff
@@ -0,0 +1,174 @@
+use sqlx::{Postgres, Transaction};
```

Comment:
If all the content of this file takes care of database only, maybe the best place for it should be a shared folder instead of `ws`, what do you think?

Reply:
Hmm I'm not sure about this. The content of the file is tightly coupled with the WebSocket Server logic and the `.proto` definitions.

Reply:
Actually not that much, I think we could move it, but I'd like to re think the error handling of it all.

Reply:
If you think that the queries and the way that errors are serialized here are coupled with the `ws` errors, then it's okay to leave this file here!